0001-typedb/01-start/03-quickstart.md

```diff
@@ -0,0 +1,668 @@
+---
+
+# Quickstart Guide
+
+## Overview
+
+This quickstart goes through the step-by-step process of creating and interacting with a TypeDB database designed for an
+[Identity and Access Management](https://en.wikipedia.org/wiki/Identity_management) solution that will be described 
+[later](04-iam-schema.md).
+
+<div class="note">
+[Important]
+We highly recommend completing this guide. Its goal is to prepare an environment for TypeDB exploration and development.
+The resulting database schema and data will be needed for example queries throughout the documentation.
+</div>
+
+## Prerequisites
+
+This quickstart takes advantage of TypeDB Studio, a desktop GUI tool for interacting with TypeDB. To proceed, please 
+install both TypeDB and TypeDB Studio:
+
+- [Install TypeDB](02-installation.md)
+- [Install TypeDB Studio](../../02-clients/01-studio.md#get-typedb-studio)
+
+IMPORTANT:
+<div class="note">
+[Important]
+It’s recommended to use TypeDB and TypeDB Studio of the same version number in order to ensure compatibility.
+</div>
+
+## Initialize database
+
+### Start TypeDB
+
+Run the following command in a terminal:
+
+```bash
+typedb server
+```
+
+After showing the TypeDB ASCII logo and logging the bootup completion time, TypeDB is ready for connections.
+
+![TypeDB server bootup message](../../images/quickstart/quickstart-launched-typedb.png)
+
+### Connect to TypeDB
+
+Launch TypeDB Studio, then:
+
+1. Click the [**Connect to TypeDB**] button (right side of the toolbar)
+2. Enter localhost:1729 in the [**Address**] field
+3. Click the [**Connect**] button (the dialog will close after a successful connection)
+
+![Connection Manager Connected](../../images/studio/connection-interface-connected.png)
+
+### Create a project
+
+TypeDB Studio projects allow you to organize and save related queries for future reuse.
+
+![Project Interface empty](../../images/quickstart/studio-projects-empty.png)
+
+To create a new project:
+
+1. Click the [**Open Project**] button in the [**Project**] panel (upper left)
+2. Choose a directory for the project files
+3. Click the [**Open**] button
+
+The [**Project**] panel will now display the root directory and a nested hidden directory for unsaved files.
+
+![Project Interface With Open Folder](../../images/studio/project-interface-created-folder.png)
+
+### Create a database
+
+1. Click the database icon to open the [**Manage Databases**] dialog (left side of the toolbar)
+
+   ![Database Manager Empty](../../images/studio/studio-database.png)
+
+2. Enter `iam` in the text field, and click the [**Create**] button next to it
+3. Click the [**Close**] button in the bottom right
+
+   ![Database Manager With Phone Calls Database](../../images/studio/databases-interface-iam-database.png)
+
+4. Select `iam` from the database dropdown (next to the database icon)
+
+### Prepare a TQL file
+
+A [TypeDB schema](../02-dev/02-schema.md) contains entity, relation, and attribute type definitions that make up the 
```

Comment:
```suggestion
A [TypeDB schema](../02-dev/02-schema.md) contains entity, relation and attribute type definitions that make up the 
```

Reply:
https://www.ox.ac.uk/

Reply:
Isn't it an Oxford comma? I believe it's almost mandatory in American English.